Engines/Wine/QuickScript/Zip Script/script.js

```diff
@@ -1,7 +1,8 @@
 const QuickScript = include("engines.wine.quick_script.quick_script");
 const Downloader = include("utils.functions.net.download");
 const Wine = include("engines.wine.engine.object");
-const { Extractor } = include("utils.functions.filesystem.extract");
+const {Extractor} = include("utils.functions.filesystem.extract");
```

Comment:
I don't get this: sometimes a space is added around `{}`, sometimes it's removed. How can this happen?

Reply:
I think visual studio code normally adds a space for better readability. While other editors seem to remove it

Reply:
This is nothing the editors should take care of. It should be eslint.

Reply:
I this eslint does not care...

Reply:
Oh it does: https://eslint.org/docs/rules/object-curly-spacing

Reply:
We need a PR where we update the eslint rules:
- https://eslint.org/docs/rules/object-curly-spacing
- the arrow lambda function rule

Reply:
Can you do it?